libcaf_net/caf/net/http/router.cpp

```diff
@@ -65,6 +65,11 @@ void router::shutdown(const error& err) {
   down_->shutdown(err);
```

Comment:
This code is not DRY. 😉
Please call the new function from the old one.

Reply:
:speak_no_evil: